### PR TITLE
refactor: transaction info is translated in the appropriate places

### PR DESCRIPTION
**Pull request by moodysalem**
fixes https://github.com/Uniswap/interface/issues/1756

### CODE REVIEW

**Comment by moodysalem** [comment]
we should also remove `pendingText` from the confirmation modal and just use a single transaction summary component as the human readable representation of a transaction

File: src/components/AccountDetails/Transaction.tsx [hunk 1]
```diff
@@ -40,26 +39,26 @@ export default function Transaction({ hash }: { hash: string }) {
...
-    <TransactionWrapper>
+    <div>
       <TransactionState
         href={getExplorerLink(chainId, hash, ExplorerDataType.TRANSACTION)}
         pending={pending}
         success={success}
       >
         <RowFixed>
-          <TransactionStatusText>{summary ?? hash} ↗</TransactionStatusText>
+          <TransactionStatusText>{<TransactionSummary info={info} />} ↗</TransactionStatusText>
```
Comment (justindomingue) [inline src/components/AccountDetails/Transaction.tsx:56]: should the arrow be added by `TransactionSummary`?

File: src/components/AccountDetails/TransactionSummary.tsx [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+  SubmitProposalTransactionInfo,
+  TransactionInfo,
+  TransactionType,
+  VoteTransactionInfo,
+  WithdrawLiquidityStakingTransactionInfo,
+  WrapTransactionInfo,
+} from '../../state/transactions/actions'
+
+function formatAmount(amountRaw: string, decimals: number, sigFigs: number): string {
+  return new Fraction(amountRaw, JSBI.exponentiate(JSBI.BigInt(10), JSBI.BigInt(18))).toSignificant(sigFigs)
```
Comment (justindomingue) [inline src/components/AccountDetails/TransactionSummary.tsx:28]: should `JSBI.BigInt(18)` be `JSBI.BigInt(decimals)`?

File: src/components/AccountDetails/TransactionSummary.tsx [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+
+  return <Trans>Approve {token?.symbol}</Trans>
+}
+
+function VoteSummary({ info }: { info: VoteTransactionInfo }) {
+  const proposalKey = `${info.governorAddress}/${info.proposalId}`
+  if (info.reason && info.reason.trim().length > 0) {
+    switch (info.decision) {
+      case VoteOption.For:
+        return <Trans>Voted in favor of proposal {proposalKey}</Trans>
```
Comment (justindomingue) [inline src/components/AccountDetails/TransactionSummary.tsx:95]: Inconsistent tense, should they all be in the past?

Comment (justindomingue) [reply src/components/AccountDetails/TransactionSummary.tsx:95]: dito below on the tense

File: src/components/AccountDetails/TransactionSummary.tsx [hunk 4]
```diff
@@ -0,0 +1,325 @@
...
+      <Trans>
+        Wrap <FormattedCurrencyAmount rawAmount={currencyAmountRaw} symbol={'ETH'} decimals={18} sigFigs={6} /> to WETH
+      </Trans>
+    )
+  }
+}
+
+function DepositLiquidityStakingSummary({}: { info: DepositLiquidityStakingTransactionInfo }) {
+  // not worth rendering the tokens since you can should no longer deposit liquidity in the staking contracts
+  // todo: deprecate and delete the code paths that allow this, show user more information
```
Comment (justindomingue) [inline src/components/AccountDetails/TransactionSummary.tsx:148]: probably good to open an issue as a refactor item 😄

File: src/state/governance/model.ts [hunk 5]
```diff
@@ -0,0 +1,5 @@
+export enum VoteOption {
```
Comment (justindomingue) [inline src/state/governance/model.ts:1]: `model` is a weird name for the file. Wouldn't we usually call this `types.ts`?

**Review by justindomingue — APPROVED** [review]

File: src/components/AccountDetails/TransactionSummary.tsx [hunk 6]
```diff
@@ -0,0 +1,325 @@
...
+      case VoteOption.For:
+        return (
+          <Trans>
+            Voted for proposal {proposalKey} with reason &quot;{info.reason}&quot;
+          </Trans>
+        )
+      case VoteOption.Abstain:
+        return (
+          <Trans>
+            Vote to abstain on proposal {proposalKey} with reason &quot;{info.reason}&quot;
```
Comment (justindomingue) [inline src/components/AccountDetails/TransactionSummary.tsx:112]:
```suggestion
            Voted to abstain on proposal {proposalKey} with reason &quot;{info.reason}&quot;
```

should this be `voted`?

**Pull request merged**